### PR TITLE
Document obl:agent for English passives

### PR DESCRIPTION
**Pull request by nschneid**
Per https://github.com/UniversalDependencies/UD_English-EWT/issues/290, English treebanks have been adopting `obl:agent`. This PR adds it to the English docs.

### CODE REVIEW

**Comment by nschneid** [comment]
Also made small tweaks to the universal docs. Will leave this open for a couple days in case @dan-zeman wants to do a sanity check before I merge.

**Comment by AngledLuffa** [comment]
Does this all need a link here

https://universaldependencies.org/u/dep/obl-agent.html

or does that get automatically generated once this is merged?

**Comment by nschneid** [comment]
`[obl:agent]()` will autolink if that's what you're asking

File: _u-dep/aux_.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ expresses categories such as tense, mood, aspect, voice or evidentiality. It is
 are also treated as instances of `aux`. 
 
 **New from v2:** Auxiliares used to construct the passive [voice](u-feat/Voice) are now also labeled `aux`,
-although we strongly encourage the use of the subtype `aux:pass` in language that have a grammaticalized (periphrastic)
+although we strongly encourage the use of the subtype [aux:pass]() in language that have a grammaticalized (periphrastic)
```
Comment (dan-zeman) [inline _u-dep/aux_.md]: in languages that have...

**Comment by dan-zeman** [comment]
> Will leave this open for a couple days in case @dan-zeman wants to do a sanity check before I merge.

Skimmed through the changes, seems good to me (except for one comment on subject-verb agreement).

**Pull request merged**